src/repl.jl

```diff
@@ -1,103 +0,0 @@
-import Base: LineEdit, REPL
```

Comment:
Sure, will do that. I'd removed all the files when I started working on this branch and I haven't gotten to the point where I put everything back in yet.

Reply:
Tested the REPL on Windows and Mac and everything seems to work. Currently the REPL works on Linux if it is manually initialized.

![screen shot 2019-01-31 at 2 43 27 pm](https://user-images.githubusercontent.com/1813121/52087424-aa908280-2566-11e9-9ef2-68c171184fdc.png)

I changed the REPL keybinding to `[` since `]` is taken by the package manager.